### PR TITLE
Added variable to enable/disable UMR tbl pack

### PR DESCRIPTION
**Pull request by hoopoepg**
- default value: UCX_IB_ENABLE_UMR_PACK=y

### CODE REVIEW

File: src/uct/ib/base/ib_md.c [hunk 1]
```diff
@@ -97,6 +97,10 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Prefer nearest device to cpu when selecting a device from NET_DEVICES list.\n",
      ucs_offsetof(uct_ib_md_config_t, ext.prefer_nearest_device), UCS_CONFIG_TYPE_BOOL},
 
+    {"ENABLE_UMR_PACK", "y",
```
Comment (yosefe) [inline src/uct/ib/base/ib_md.c]: "INDIRECT_ATOMIC"

File: src/uct/ib/base/ib_md.c [hunk 2]
```diff
@@ -97,6 +97,10 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
...
+     "Enable UMR pack\n",
```
Comment (yosefe) [inline src/uct/ib/base/ib_md.c]: "Use indirect memory key for atomic operation"

File: src/uct/ib/base/ib_md.c (hunk 1 above)
Comment (hoopoepg) [reply src/uct/ib/base/ib_md.c]: renamed

File: src/uct/ib/base/ib_md.c (hunk 2 above)
Comment (hoopoepg) [reply src/uct/ib/base/ib_md.c]: description is updated

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7420/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7421/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
@yosefe anything else to fix?

**Pull request merged**